frontend/src/pages/profile/profileDetailView.tsx

```diff
@@ -12,6 +13,11 @@ export const ProfileDetailView: React.FC<Props> = ({ history }) => {
         <IonButton routerLink="/profile/changepassword" color="primary">
           Password ändern
         </IonButton>
+        <IonButton
+          onClick={() => Auth.signOut().then(console.log).catch(console.error)}
+        >
+          Ausloggen
+        </IonButton>
```

Comment:
```suggestion
        <IonButton
          onClick={Auth.signOut}
        >
          Ausloggen
        </IonButton>
```
I guess this should work to for logout